cms/db/contest.py

```diff
@@ -251,6 +251,10 @@ class Contest(Base):
         Interval,
         CheckConstraint("min_submission_interval > '0 seconds'"),
         nullable=True)
+    ignore_submission_rate_limit_period = Column(
```

Comment:
This setting is closely related to the above setting, so I'd make the naming similar.

What about `min_submission_interval_grace_period`?